app/src/main/res/xml/pref_input.xml

```diff
@@ -80,6 +80,13 @@
             app:summaryOff="@string/pref_mouse_passthrough_summary_off"
             app:summaryOn="@string/pref_mouse_passthrough_summary_on"
             app:title="@string/pref_mouse_passthrough" />
+
+        <SwitchPreference
+            app:defaultValue="true"
+            app:key="hide_local_cursor"
+            app:summaryOff="@string/pref_hide_local_cursor_summary_off"
+            app:summaryOn="@string/pref_hide_local_cursor_summary_on"
```

Comment:
Summary is unnecessary here, as preference title is clear enough IMO. 
We can even use `Hide Local (Android) Cursor` as title to make it more clear if you want.